coderedcms/templatetags/coderedcms_tags.py

```diff
@@ -67,8 +67,11 @@ def get_pictures(collection_id):
 
 
 @register.simple_tag
-def get_navbars():
-    return Navbar.objects.all()
+def get_navbars(request):
+    curr_url = request.site.hostname
+    site = Site.objects.get(hostname=curr_url)
+    layout = LayoutSettings.objects.get(site_id=site)
+    return Navbar.objects.filter(layoutsetting_id=layout.id)
```

Comment:
Ditto comment above. If the `Navbar` were to have a foreign key to a `Site`, you could eliminate the LayoutSettings here and filter the navbar directly by the site, e.g.:

```python
Navbar.objects.filter(site=site)
```